projects/netgrif-components-core/src/lib/data-fields/i18n-field/models/i18n-field.ts

```diff
@@ -4,15 +4,181 @@ import {Layout} from '../../models/layout';
 import {Validation} from '../../models/validation';
 import {Component} from '../../models/component';
 import {I18nFieldValue} from './i18n-field-value';
+import {Observable} from 'rxjs';
+import {FormControl, ValidatorFn} from '@angular/forms';
 
+export enum I18nFieldValidation {
+    TRANSLATION_REQUIRED = 'translationRequired',
+    TRANSLATION_ONLY = 'translationOnly',
+    REQUIRED_I18N = 'requiredI18n'
+}
+
+export const DEFAULT_LANGUAGE_CODE = 'xx';
 
 export class I18nField extends DataField<I18nFieldValue> {
 
+    public static toObject(templateValue: I18nFieldValue): any {
```

Comment:
should return at least an `object` instead of `any`. A more narrowly defined type would be preferred still.

Reply:
Done